schema.graphql

```diff
@@ -1,18 +1,17 @@
 type Domain @entity {
-  id: ID!
-  labelhash: Bytes
-  parent: Domain
-  subdomains: [Domain!]! @derivedFrom(field: "parent")
-  subdomainCount: Int!
+  id: ID!               # The namehash of the name
+  labelName: String     # The human readable label name (imported from CSV), if known
+  labelhash: Bytes      # keccak256(labelName)
+  parent: Domain        # The namehash (id) of the parent name
+  subdomains: [Domain!]! @derivedFrom(field: "parent")  # Can count domains from length of array
   owner: Account!
   resolver: Bytes
-  ttl: Int
+  ttl: BigInt
 }
 
 type Account @entity {
   id: ID!
-  domains: [Domain!]! @derivedFrom(field: "owner")
-  domainCount: Int!
+  domains: [Domain!]! @derivedFrom(field: "owner") # Can count domains from length of array
```

Comment:
Isn't counting them `O(n)`? This could get slow in the case of, say, counting all `.eth` domains.

Reply:
Good point, I will revert to the previous way with `domainCount`